### PR TITLE
[New Rule] UAC Bypass via Windows Firewall Snap-in Hijack

### PR DESCRIPTION
**Pull request by Samirbous**
## Issues

Resolves https://github.com/elastic/detection-rules/issues/375

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

**Comment by Samirbous** [comment]
Alert Example:

image

File: rules/windows/privilege_escalation_uac_bypass_winfw_mmc_hijack.toml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+creation_date = "2020/10/14"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/10/14"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies User Account Control (UAC) bypass via hijacking the Windows Firewall Snap-in. Attackers bypass UAC to
+stealthily execute code with elevated permissions.
```
Comment (threat-punter) [inline rules/windows/privilege_escalation_uac_bypass_winfw_mmc_hijack.toml:11]:
```suggestion
Identifies attempts to bypass User Account Control (UAC) by hijacking the Microsoft Management Console (MMC) Windows
Firewall snap-in. Attackers bypass UAC to stealthily execute code with elevated permissions.
```

File: rules/windows/privilege_escalation_uac_bypass_winfw_mmc_hijack.toml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "UAC Bypass via Windows Firewall Snap-in Hijack"
```
Comment (threat-punter) [inline rules/windows/privilege_escalation_uac_bypass_winfw_mmc_hijack.toml:17]:
```suggestion
name = "UAC Bypass via Windows Firewall Snap-In Hijack"
```

File: rules/windows/privilege_escalation_uac_bypass_winfw_mmc_hijack.toml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+severity = "medium"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Privilege Escalation"]
+type = "eql"
+
+query = '''
+process where event.type in ("start", "process_started") and
+ process.parent.name == "mmc.exe" and
+ /* process.Ext.token.integrity_level_name == "high" can be added in future for tuning */
+ /* args of the Windows Firewall SnapIn */
+  process.parent.args == "WF.msc" and process.name!="WerFault.exe"
```
Comment (threat-punter) [inline rules/windows/privilege_escalation_uac_bypass_winfw_mmc_hijack.toml:30]:
```suggestion
  process.parent.args == "WF.msc" and process.name != "WerFault.exe"
```

**Review by threat-punter — APPROVED** [review]
Looks great. I made a few minor suggestions 👍

**Review by bm11100 — APPROVED** [review]

**Pull request merged**